### ISSUE TITLE
2020-11-28: Ending of verbs in Japanese translations

### ISSUE THREAD

**Issue by naoe-tatara** [carpentries/glosario#229]
I wonder if there is any rule on which ending style of verbs, either so-called 'desu-masu' or 'dearu', to be used in Japanese translations. It seems that translations of the lesson materials are in 'desu-masu' style. So should translations in Glosario also in 'desu-masu' style? Thank you for your advice in advance.

**Comment by masamiy** [comment]
Good point! and sorry for the late response. As you pointed out that Japanese lessons are in 'desu-masu' style, I personally think it may be easy to keep the same style in Glosario, too. Though it may be too polite as Glossary, consistency would make translators' life a little easier. What do you think?

**Comment by naoe-tatara** [comment]
Thank you for the answer! I agree with you @masamiy on the importance of consistency. Meanwhile, especially in Glossario, I feel "taigen dome"-style (or "... no koto", for example) should work best sometimes rather than sticking to "desu-masu" style. . I don't think these may be too inconsistent with "desu-masu" style and sometimes easier to read.

**Comment by masamiy** [comment]
Thank you for sharing your thought! So, if I re-write this function definition with a new form, will it be:
"別の処理の中で使用する、あるいは繰り返し使用するために、連続した操作をひとかたまりの
      コードブロックとして名前を付けて定義したもの。関数は任意の引数を使って、
      入力ともしあれば返された出力(返り値)を定義する。関数は定義されたタスクを、
      関数呼び出しと呼ばれる１つのコマンドで繰り返し使用することを可能にする。”
？

I will ask at Carpentries Japan Slack (carpentries-ja.slack.com) to see how they feel if it is okay with you (though there are only a few active members ;)).

**Comment by naoe-tatara** [comment]
No, i think it's fine with the current version with "desu-masu" ending and no need to change to "dearu", as we can keep consistency. Probably "taigen dome" is nothing conflicting with "desu-masu" style, and I may have confused you. In case of "function", the first sentence ends with "のこと", but not "のことです", which for me is natural. Similarly, sometimes we will be having cases where it sounds natural to end with noun rather than adding "desu" after the noun. In such cases we don't need to stick to ending with "desu". This is what I meant. Hope it makes sense ☺️

**Comment by masamiy** [comment]
Thank you for clarifying. So, do you think as long as the a sentence with defining a meaning does not use 'desu-masu', would it be okay?
A one person has a strong preference to taigen dome style and I thought it would very different from desu-masu!
I will translate one entry for my practice and ask you to have a look to see if it follows the style. Would it be okay??

**Comment by naoe-tatara** [comment]
I see, thank you for your opinion. I found that taigen-dome is used often with desu-masu style, so I did not feel that it is very different. So the current version of the translation of "function" is fine, I think. But do you on the other hand feel that the first sentence should end with "定義したものです" rather than the current "定義したもの"? I translated recently "repository", and used taigen-dome style; "[バージョン管理システム](#version_control_system)において、管理対象となるプロジェクトを構成するファイルとその履歴を説明するメタデータを保存する場所。" I thought it is a bit too much to end with "...を保存する場所です。" But I will take a look at your PR :)

**Comment by masamiy** [comment]
I think your 'repository' translation is perfect! Do you think that "avoid 'desu-masu' style in Glosario" would be a good enough guideline? Usually dictionaries are written in less polite manner and we had a Slack discussion about Glosario style: https://carpentries-ja.slack.com/archives/C01H1AY3C7K/p1607650781000800 
I think avoiding 'desu-masu' style throughout entries is just easy enough to be consistent. What do you think?

**Comment by naoe-tatara** [comment]
Yes, I totally agree with you; "avoid 'desu-masu' style in Glosario" is a perfect solution 😃

**Comment by masamiy** [comment]
Great! I would like to add this short sentence somewhere in Glosario as I mentioned in Slack channel. If you have an idea, please let me or @TomKellyGenetics know. If you are happy with our solution, please feel free to close this issue (unless it is something I should do :)

**Comment by TomKellyGenetics** [comment]
I'm not familiar with other styles (except crude 方言) so bear with me. I looked up 体言止め and it seems to be appropriate.

+1 for consistency going forward. The question remaining I think it how to make this clear to new contributors so they don't feel they made a mistake if they submit a である definition.

**Comment by masamiy** [comment]
I personally do not think it is a good idea to just state 'Use 体言止め' as a style guideline.
 
 My understanding of Taigen dome is to end a sentence with noun. So, the first sentence of each entry can be 体言止め as usually it defines the meaning, but I am not sure if taigen dome is appropriate in any situations.

For example, if I make all sentences in 'function' 体言止め, it would be:
"1. 別の処理の中で使用する、あるいは繰り返し使用するために、連続した操作をひとかたまりの
コードブロックとして名前を付けて定義したもの。
2. 関数は任意の引数を使って、入力ともしあれば返された出力(返り値)を定義。
3. 関数は定義されたタスクを、
関数呼び出しと呼ばれる１つのコマンドで繰り返し使用することが可能。”

1. and 3. may be okay, but to me, the second sentence sounds more natural if it says:
"... 関数は任意の引数を使って、入力ともしあれば返された出力(返り値)を定義する。",
which is not taigen-dome style. 

So, I think just state "do not use desu-masu style" would be sufficient enough style guideline.

**Comment by naoe-tatara** [comment]
Sorry for not being able to comment here earlier. I agree with @masamiy here; 2 sounds more natural when it ends like "...を定義する。"

On the other hand, I got probably a bit confusion and would like to make sure my understanding is in line with you all:
Regarding consistency of translation style, I first thought it is best to keep consistency everywhere Japanese translation concerns. I raised this issue, because "taigen-dome" in combination with "dearu" ending style is (AFAIK) most often used in glossaries or dictionaries, while I was aware that lesson translations used polite "desu-masu" style. But when I looked back the translation of "function", I thought probably "taigendome" in combination with "desu-masu" style is OK if we pursue consistency throughout the translated materials in Carpentries. After that, "avoid 'desu-masu' style" is suggested, which I thought is good and close to my original opinion, but that really means then use of "dearu" style instead where "desu-masu" would apply otherwise. So is it correct so that we are talking about consistency *within* Glosario?

Long story (sorry) short, IMO 
1) consistency within Glosario is essential. 
2) sounding natural in context of glossary-type of documents compared to common ones would outweigh using "desu-masu" style everywhere, and 
3) "In Glosario, Japanese translation should avoid "desu-masu" style and write in "dictionary"-style" is reasonable.

**Comment by masamiy** [comment]
Thanks @naoe-tatara for summarising our very deep discussion! I believe that Japanese lessons can stay as it is, i.e., with 'desu-masu' style. However, as Glosario is glossaries, dictionary form (i.e., plain form or casual form) is more appropriate. So, my understanding is that as long as your Japanese translation sounds like Wikipedia, it is fine...?

At this stage, there are only a few of us working on Glosario Japanese, why do not we have a go with "Japanese translation should avoid "desu-masu" style and write in "dictionary"-style"? If a new comer have a confusion with this guideline, we can always add a new line and make it better.

**Comment by elletjies** [comment]
@all-contributors please add @naoe-tatara question
@all-contributors please add @TomKellyGenetics  question
@all-contributors please add @masamiy  question

**Comment by elletjies** [comment]
@all-contributors please add @TomKellyGenetics question

**Comment by elletjies** [comment]
@all-contributors please add @masamiy question

**Comment by elletjies** [comment]
@all-contributors please add @naoe-tatara question
